src/main/java/com/labuhn/minesweeper/application/Application.java

```diff
@@ -0,0 +1,5 @@
+package com.labuhn.minesweeper.application;
```

Comment:
Would be great to have this file in our root folder minesweeper. Please revert this change